modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/ClassicEditor.js

```diff
@@ -107,12 +92,6 @@ const ClassicEditor = React.forwardRef(
 			};
 		}, [contents, getHTML, name]);
 
-		const onResize = debounce(() => {
```

Comment:
I need to checkout the PR to see if resize still working :p

Reply:
Resize actually does not work on master :disappointed: 

This code is leftover from times when we changes toolbars on different sizes, for example had a smaller one on mobile. It should be safe to remove.

Reply:
Ok!